soc/arm/rpi_pico/rp2/Kconfig.soc

```diff
@@ -0,0 +1,26 @@
+# Raspberry Pi RP2XXX MCU line
+
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# Copyright (c) 2021 Yonatan Schachter
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "RP2xxx MCU Selection"
+	depends on SOC_SERIES_RP2XXX
+
+config SOC_RP2040
+	bool "Raspberry Pi RP2040"
+
+endchoice
+
+config RP2_FLASH_W25Q080
+	bool
+
+config RP2_FLASH_GENERIC_03H
+	bool
+
+config RP2_FLASH_IS25LP080
+	bool
+
+config RP2_FLASH_W25X10CL
+	bool
```

Comment:
can these options include help? are they part of a choice?

Reply:
I made:
https://github.com/rivimey/zephyr/blob/0cf92bc279f1b5c68d615f3c2457210d915a11f4/soc/arm/rpi_pico/rp2/Kconfig.soc

[also needs a tweak in  modules/hal_rpi_pico/CMakeLists.txt to supply the missing bootloader assembly filename.]